src/main/java/com/fundynamic/d2tm/game/map/Cell.java

```diff
@@ -66,7 +59,7 @@ public Entity getEntity() {
 
     public void setEntity(Entity entity) {
         if (this.entity != null) {
-            throw new IllegalArgumentException("Cannot place mapEntity (" + entity + ") because mapEntity already exists: " + this.entity);
+            throw new CellAlreadyOccupiedException("Cannot place Entity (" + entity + ") on cell because Entity already present: " + this.entity);
```

Comment:
Maybe I'm thinking to far ahead, but how will this work for infantry? We should be able to have multiple units in a single cell...